src/containers/UserSettings/UserSettings.tsx

```diff
@@ -33,6 +39,21 @@ function UserSettings(props: any) {
         props.setSettingValue(USE_NODES_ENDPOINT_IN_DIAGNOSTICS_KEY, String(value));
     };
 
+    const renderBreakNodesSettingsItem = (title: ReactNode) => {
+        return (
+            <div className={b('item-with-popup')}>
+                {title}
+                <HelpPopover
+                    content={
+                        'Use /viewer/json/nodes endpoint for Nodes Tab in diagnostics. It returns incorrect data on older versions'
+                    }
```

Comment:
this could be just a string prop?

Reply:
I don't understand your question. Could you clarify please?

Reply:
`content="Use ..."`
vs
`content={'Use ....'}`
it's not that important